tslint-server/src/server.ts

```diff
@@ -71,6 +71,7 @@ namespace StatusNotification {
 	export const type = new server.NotificationType<StatusParams, void>('tslint/status');
 }
 
+let rootDir: string | null = null;
```

Comment:
Looks like I had an unstaged change :( I'll update the PR, in case you still want to accept it.